TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/IntakeRunningCode.java

```diff
@@ -3,13 +3,19 @@
 import com.qualcomm.robotcore.eventloop.opmode.OpMode;
 import com.qualcomm.robotcore.hardware.DcMotor;
 
+/** Defines Class IntakeRunningCode. */
 public class IntakeRunningCode extends OpMode{
+    /** Defines a Motor called intakeMotor. */
     DcMotor intakeMotor;
+
+    /** Adds intakeMotor to the HardwareMap. */
```

Comment:
You generally don't have to comment overridden methods, Javadoc knows to use the documentation from the parent class.